src/utils/getNeededDates.js

```diff
@@ -13,12 +13,9 @@ function getNeededDates (year, month) {
     const endDate = new Date(year, month, 0);
     endDate.setHours(23, 59, 59, 999);
 
-    const todayStart = new Date(Date.now());
-    todayStart.setHours(0, 0, 0, 0);
-
     // Check who need to know about local time, and then pass this variables?
-    const localStartDate = startDate.toLocaleString();
-    const localEndDate = endDate.toLocaleString();
+    // const localStartDate = startDate.toLocaleString();
+    // const localEndDate = endDate.toLocaleString();
 
     return { startDate, endDate, todayStart };
```

Comment:
Isn't this returning an error?

Reply:
No, i just commented redundant operations. This code is to remmember how we can create dates with local time.